tests/components/songpal/test_config_flow.py

```diff
@@ -243,7 +223,7 @@ async def test_import_invalid(hass):
             DOMAIN, context={"source": SOURCE_IMPORT}, data=CONF_DATA
         )
         assert result["type"] == RESULT_TYPE_ABORT
-        assert result["reason"] == "connection"
+        assert result["reason"] == "cannot_connect"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)